ios/RCCNavigationController.m

```diff
@@ -249,6 +249,8 @@ - (void)performAction:(NSString*)performAction actionParams:(NSDictionary*)actio
     }
     
     BOOL animated = actionParams[@"animated"] ? [actionParams[@"animated"] boolValue] : YES;
+
+    [self.viewControllers.lastObject viewWillDisappear:animated];
```

Comment:
For safety, could you validate size of self.viewControllers?

Reply:
OK!